radio/src/targets/horus/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ option(AFHDS3 "AFHDS3 TX Module" ON)
 option(MULTIMODULE "DIY Multiprotocol TX Module (https://github.com/pascallanger/DIY-Multiprotocol-TX-Module)" ON)
 option(GHOST "Ghost TX Module" ON)
 option(HARDWARE_EXTERNAL_ACCESS_MOD "Support for R9M 2019 hardware mod" OFF)
+option(SPACEMOUSE "Support for SpaceMouse" ON)
```

Comment:
Should this not be OFF by default to save cluttering the UI for the majority?

Reply:
That is a question of how we want to proceed. What do you suggest @pfeerick ?

Reply:
I can work with the serial port setting being used to determine whether the 6 sources are visible to save adding the build option.

Reply:
I have no issue generally with stuff being on by default - with the build option being there to remove that feature if tight on memory, debugging, or need to create a 'lite' or feature limited firmware build. In this instance, I think we need options on the hardware page that allow you to turn the mods *on* for the uncommon ones, thus enabling their visibility in the other pages. As the reality is we shouldn't have this mod on-by-default for the five people out of 5000 that might do the mod ;)